extensions/pyRevitTools.extension/pyRevit.tab/Selection.panel/memo.stack3/Paste State.pushbutton/script.py

```diff
@@ -383,11 +384,14 @@ def set_tansform_matrix(selvp, selview):
         f = open(datafile, 'r')
         id = pickle.load(f)
         f.close()
+        selected_elements = revit.get_selection().elements
         with revit.Transaction('Paste Visibility Graphics'):
-            revit.activeview.ApplyViewTemplateParameters(
-                revit.doc.GetElement(DB.ElementId(id))
-                )
-    except Exception:
+            for view in copy_paste_state_utils.get_views(selected_elements):
+                view.ApplyViewTemplateParameters(
+                    revit.doc.GetElement(DB.ElementId(id))
+                    )
+    except Exception as exc:
+        logger.debug(exc)
```

Comment:
_here and below_ I've added a debug line, because other exception than IO exceptions. 

I had an exception when Crop region couldn't be set because of a custom shape.

Maybe it makes sense to show it with `warn`